src/views/GlossaryPage.vue

```diff
@@ -0,0 +1,279 @@
+    <DashboardCard>
+      <template #title> Agencies and Organizations </template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.95rem">
+          <p v-for="agency in agencyAcronyms" :key="agency.acronym">
+            <strong style="font-size: 1.02rem">{{ agency.acronym }}:</strong>
+            {{ agency.name }}
+          </p>
+        </div>
+      </template>
+    </DashboardCard>
+    <DashboardCard>
+      <template #title>Models and Data</template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.95rem">
+          <p v-for="model in modelAcronyms" :key="model.acronym">
+            <strong style="font-size: 1.02rem">{{ model.acronym }}:</strong>
+            {{ model.name }}
+          </p>
+        </div>
+      </template>
+    </DashboardCard>
+    <DashboardCard>
+      <template #title>Data Variables</template>
+      <template #content>
+        <div class="px-4" style="font-size: 0.9rem">
+          <p v-for="data in dataAcronyms" :key="data.variable">
+            <strong style="font-size: 1.02rem">{{ data.variable }}:</strong>
+            ({{ data.units }})
+            {{ data.description }}
+          </p>
+        </div>
+      </template>
+    </DashboardCard>
+  </DashboardLayout>
+</template>
+
+<script setup lang="ts">
+import DashboardLayout from "@/layouts/dashboard.vue";
+import DashboardCard from "@/components/base/DashboardCard.vue";
+import { sortByProperty } from "../utils/utils";
+
+const agencyAcronyms = [
+  { acronym: "RIDDC", name: "Rhode Island Data Discovery Center" },
+  {
+    acronym: "RI C-AIM",
+    name: "Rhode Island Consortium for Coastal Ecology Assessment, Innovation, and Modeling",
+  },
+  {
+    acronym: "RIDEM-OWR",
+    name: "Rhode Island Department of Environmental Monitoring-Office of Water Resources",
+  },
+  { acronym: "NBFSMN", name: "Narragansett Bay Fixed Site Monitoring Network" },
+  {
+    acronym: "URI-GSO",
+    name: "University of Rhode Island, Graduate School of Oceanography",
+  },
+  { acronym: "NBC", name: "Narragansett Bay Commission" },
+  { acronym: "RWU", name: "Roger Williams University" },
+  { acronym: "NBEP", name: "Narragansett Bay Estuary Program" },
+  { acronym: "URI-CI", name: "University of Rhode Island-Coastal Institute" },
+
+  {
+    acronym: "MassDEP",
+    name: "Massachusetts Department of Environmental Protection",
+  },
+  { acronym: "SNEP", name: "Southern New England Estuary Program" },
+  {
+    acronym: "NERACOOS",
+    name: "Northeast Regional Association Coastal Ocean Observing Systems",
+  },
+];
+
+const modelAcronyms = [
+  {
+    acronym: "ERDDAP",
+    name: "Environmental Research Division Data Access Program",
+  },
+  { acronym: "OSOM", name: "Ocean State Ocean Model" },
+  { acronym: "ROMS", name: "Regional Ocean Modelling System" },
+  { acronym: "NABATS", name: "Narragansett Bay Time Series" },
+];
+
+const dataAcronyms = [
+  {
+    variable: "ChlorophyllSurface",
```

Comment:
as this is a hand-rolled definition list, I'd consider dropping all of the bottom/surface variants and include just the base measurement